components/TokenInput.tsx

```diff
@@ -0,0 +1,44 @@
+import TokenSelectionModal from "@/components/TokenSelectionModal";
+import { useToken } from "@/contexts/TokenContext";
+import { TokenListContextItem } from "@/contexts/TokenContext";
+import { Input } from "@chakra-ui/input";
+import { useDisclosure } from "@chakra-ui/react";
+import { useState } from "react";
+
+type TokenInputProps = {
+  onSelect: (token: TokenListContextItem) => void; // eslint-disable-line no-unused-vars
```

Comment:
if we extend from an eslint typescript-specific config, then this eslint-disable-line should go away after a vscode window reload.

.eslint.config.json
```ts
{
  "extends": [
    "next/core-web-vitals",
    "eslint:recommended",
    "plugin:@typescript-eslint/eslint-recommended"
  ]
}

Reply:
Seems to still pop up errors when the plugin is added. I will look at other configs

Reply:
Wasn't able to fix quickly will add back the comments and made a ticket

Reply:
sounds good!